x-pack/plugins/security_solution/public/exceptions/pages/shared_lists/index.tsx

```diff
@@ -83,7 +84,9 @@ export const SharedLists = React.memo(() => {
   const [referenceModalState, setReferenceModalState] = useState<ReferenceModalState>(
     exceptionReferenceModalInitialState
   );
-  const [filters, setFilters] = useState<ExceptionListFilter | undefined>(undefined);
+  const [filters, setFilters] = useState<ExceptionListFilter | undefined>({
+    types: [ExceptionListTypeEnum.DETECTION, ExceptionListTypeEnum.ENDPOINT],
+  });
```

Comment:
@WafaaNasr @peluja1012 This PR adds a filter to only display shared lists and the endpoint list.

Reply:
Thanks Devin 🥳